starter_code/app.js

```diff
@@ -9,10 +9,40 @@ const punkAPI = new PunkAPIWrapper();
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
+hbs.registerPartials(__dirname + '/views/partials');
+
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
 
-app.listen(3000);
\ No newline at end of file
+// Route beer
+app.get('/beer', (req, res, next) => {
+  const beers = punkAPI.getBeers()
+  .then(beers => {
+    res.render('beer', {beers});
+  })
+  .catch(error => {
+    console.log(error)
+  })
+
+  
+});
+
+//Route randombeer
+app.get('/random-beer', (req, res, next) => {
+  const randombeer= punkAPI.randomBeer()
```

Comment:
The method is:
getRandom();

In other hand, you can clean your code like this:
```js
app.get("/random-beer", (req, res, next) => {
	punkAPI
		.getRandom()
		.then(randombeer => {
			res.render("random-beer", randombeer);
		})
		.catch(error => {
			console.log(error);
		});
});
```